website/docusaurus.config.js 

```diff
@@ -0,0 +1,75 @@
+/**
```

Comment:
This file might not be treated as a JS file with a `.js` extension. It could be an extensionless file with the name "docusaurus.config.js".

Reply:
Good call, corrected